### PR TITLE
fix: update icons to paths

### PR DESCRIPTION
**Pull request by rossmoody**
- [ ] **BREAKING CHANGE?** 

## Description

This PR amends 16 icons to paths instead of stroke to stop them from disappearing when a user zooms.

12 and 16px versions of:

- `copy-stroke`
- `email-stroke`
- `table-stroke`
- `translation-updated-stroke`
- `translation-deleted-stroke`
- `translation-created-stroke`
- `translation-outdated-stroke`
- ` translation-exists-stroke`

## Checklist

* [ ] :ok_hand: SVG updates are Garden Designer approved (add the
  designer as a reviewer)
* [ ] :globe_with_meridians: SVG demo is up-to-date (`yarn start`)
* [ ] :black_medium_small_square: Renders as expected in "dark" mode
* [ ] :white_large_square: Renders as expected @ 2x scale

### CODE REVIEW

**Review by ginnywood — APPROVED** [review]
Looks good to me. @rossmoody are you going to update the Sketch symbols too? It’d be good if those matched this treatment.

**Comment by rossmoody** [comment]
@ginnywood All I did in this case was export the icons with the strokes outlined, but when I opened the icons file in Sketch they were already outlined so made the job pretty easy.

File: src/12/translation-updated-stroke.svg [hunk 1]
```diff
@@ -1,6 +1,3 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" focusable="false" viewBox="0 0 12 12">
-  <g fill="none" stroke="currentColor">
-    <rect width="6" height="8" x=".5" y=".5" rx=".5" ry=".5"/>
-    <path stroke-linecap="round" stroke-linejoin="round" d="M2.5 6.5c2-1 2-4 2-4h-2m0 1.5a3.22 3.22 0 002 2.5M3.5 2v.5m5 8h2M8.62 7.48h2m-.12-.98l1 1-1 1m-2 1l-1 1 1 1"/>
-  </g>
+<svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" fill="none" focusable="false" viewBox="0 0 12 12">
```
Comment (jzempel) [inline src/12/translation-updated-stroke.svg:1]: The `fill="none"` attribute should be removed from this line.

File: src/12/translation-deleted-stroke.svg [hunk 2]
```diff
@@ -1,6 +1,3 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" focusable="false" viewBox="0 0 12 12">
-  <g fill="none" stroke="currentColor">
-    <rect width="6" height="8" x=".5" y=".5" rx=".5" ry=".5"/>
-    <path stroke-linecap="round" stroke-linejoin="round" d="M2.5 6.5c2-1 2-4 2-4h-2m0 1.5a3.22 3.22 0 002 2.5M3.5 2v.5m5 6l3 3m0-3l-3 3"/>
-  </g>
+<svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" fill="none" focusable="false" viewBox="0 0 12 12">
```
Comment (jzempel) [inline src/12/translation-deleted-stroke.svg:1]: The `fill="none"` attribute should be removed from this line.

File: src/12/translation-created-stroke.svg [hunk 3]
```diff
@@ -1,6 +1,3 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" focusable="false" viewBox="0 0 12 12">
-  <g fill="none" stroke="currentColor">
-    <rect width="6" height="8" x=".5" y=".5" rx=".5" ry=".5"/>
-    <path stroke-linecap="round" stroke-linejoin="round" d="M2.5 6.5c2-1 2-4 2-4h-2m0 1.5a3.22 3.22 0 002 2.5M3.5 2v.5M8 10l1 1 2.5-2.5"/>
-  </g>
+<svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" fill="none" focusable="false" viewBox="0 0 12 12">
```
Comment (jzempel) [inline src/12/translation-created-stroke.svg:1]: The `fill="none"` attribute should be removed from this line.

**Review by jzempel — CHANGES_REQUESTED** [review]
Looks like a few random `fill` attributes crept in.

**Review by jzempel — APPROVED** [review]

**Pull request merged**